src/Uno.UWP/ApplicationModel/Chat/ChatMessage.Android.cs

```diff
@@ -0,0 +1,26 @@
+﻿#if __ANDROID__
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Windows.ApplicationModel.Chat
+{
+	public partial class ChatMessage : IChatItem

```

Comment:
Please follow the indentation configuration. You can use the cleanup tooling that visual studio included in the latest versions, or use the standard reformatting.